### PR TITLE
Add PDB Chains description tooltip

### PR DESCRIPTION
**Pull request by ersinciftci**
screen shot 2017-07-27 at 5 06 32 pm

### CODE REVIEW

File: src/shared/components/proteinChainPanel/ProteinChainPanel.tsx [hunk 1]
```diff
@@ -221,6 +246,17 @@ export default class ProteinChainPanel extends React.Component<ProteinChainPanel
                     <div style={{
                         marginLeft:this.props.geneXOffset || 0,
                     }}>
+                        <div style={{display: this.isExpanded ? "inherit" : "none", position: "absolute", left: 20}}>PDB
```
Comment (onursumer) [inline src/shared/components/proteinChainPanel/ProteinChainPanel.tsx]: can we put the `PDB` text in a new line?

File: src/shared/components/proteinChainPanel/ProteinChainPanel.tsx [hunk 2]
```diff
@@ -221,6 +246,17 @@ export default class ProteinChainPanel extends React.Component<ProteinChainPanel
...
+                            <DefaultTooltip
+                                placement="left"
+                                overlay={this.helpTooltipContent}
+                                arrowContent={<div className="rc-tooltip-arrow-inner"/>}
+                                destroyTooltipOnHide={true}
+                            >
+                                <i className="fa fa-info-circle" style={{paddingLeft: 5}} />
+                            </DefaultTooltip>
+                            <br/>Chains
```
Comment (onursumer) [inline src/shared/components/proteinChainPanel/ProteinChainPanel.tsx]: `Chains` in a new line?

File: src/shared/components/proteinChainPanel/ProteinChainPanel.tsx [hunk 3]
```diff
@@ -221,6 +246,17 @@ export default class ProteinChainPanel extends React.Component<ProteinChainPanel
                     <div style={{
                         marginLeft:this.props.geneXOffset || 0,
                     }}>
+                        <div style={{display: this.isExpanded ? "inherit" : "none", position: "absolute", left: 20}}>PDB
+                            <DefaultTooltip
+                                placement="left"
+                                overlay={this.helpTooltipContent}
+                                arrowContent={<div className="rc-tooltip-arrow-inner"/>}
```
Comment (onursumer) [inline src/shared/components/proteinChainPanel/ProteinChainPanel.tsx]: actually we don't need the `arrowContent` anymore, it is already in the `DefaultTooltip` library

File: src/shared/components/proteinChainPanel/ProteinChainPanel.tsx [hunk 4]
```diff
@@ -210,6 +210,31 @@ export default class ProteinChainPanel extends React.Component<ProteinChainPanel
         });
     }
 
+    public helpTooltipContent()
+    {
+        return (
+            <div style={{maxWidth: 400}}>
+                This panel displays a list of PDB chains for the corresponding uniprot ID.
```
Comment (alisman) [inline src/shared/components/proteinChainPanel/ProteinChainPanel.tsx]: this should really be done with <p> tags, but not a big deal.

**Pull request merged**